forloop_modules/function_handlers/database_handlers.py

```diff
@@ -424,13 +416,18 @@ def execute(self, node_detail_form):
         # else:
         #     raise HTTPException(status_code=response.status_code, detail="Error requesting new node from api")
         
-    def direct_execute(self, db_name, dbtable_name, selected_columns, column_name, operator, value, limit, new_var_name):        
-        db_table = get_db_table_from_db(table_name=dbtable_name, db_name=db_name)
+    def direct_execute(self, db_name, db_table_name, select, where_column_name, where_operator, where_value, limit, new_var_name):
+        db_name = parse_comboentry_input(input_value=db_name)
+        db_table_name = parse_comboentry_input(input_value=db_table_name)
+        select = parse_comboentry_input(input_value=select)
+        where_column_name = parse_comboentry_input(input_value=where_column_name)
```

Comment:
Why is it in direct execute - that sounds wrong

Reply:
I will move it to `execute` @DovaX

Reply:
Yes, please do it

Reply:
Done @DovaX